docs/user_guide/disconnected/index.md

```diff
@@ -4,286 +4,52 @@ title: Disconnected
 sidebar: sidebars/documentation.html
 ---
 
-# Katello Disconnected Utility
+# Katello Inter-Server Sync
 
 ## Intro
 
-If you are working in an environment where you wish to synchronize content from Red Hat's CDN and make available to your Katello servers that do not have a connection to the internet we have a utility that assists with this, known as katello-disconnected.  A diagram of the content flow is as follows:
+If you are working in an air-gapped network environment where some of your
+Katello servers do not have Internet connectivity, you may be interested in
+using the Katello Inter-Server Sync feature. This allows you to export repos
+(including repos in content views) on your "upstream" Katello server, and then
+import said repos into your "downstream" server that does not have
+connectivity.
 
-![Disconnected Workflow](./katello-disconnected.png)
+It is possible to export either individual yum repos, or all yum repos in a
+particular content view. For now, only yum content is supported. Future
+releases may include support for more content types such as Puppet modules.
 
-*katello-disconnected* is used via a command line interface with a summary outlined here:
+![ISS Workflow](./ISS.png)
 
+The diagram above shows an example scenario where a user wants to export all
+Yum content in a content view and then import to another Katello server. The
+ISO file is burned to media and then walked across the air-gap in the network.
 
-```
-katello-disconnected(1)        Katello Reference       katello-disconnected(1)
+NOTE: This feature is intended to replace the 'katello-disconnected' script.
```

Comment:
Should this be at the top to immediately grab users who are in fact wondering if this is the thing they should be doing instead of disconnected?